pgp/keysource.go

```diff
@@ -133,7 +133,8 @@ func (d GnuPGHome) Import(armoredKey []byte) error {
 	args := []string{"--batch", "--import"}
 	err, _, stderr := gpgExec(d.String(), args, bytes.NewReader(armoredKey))
 	if err != nil {
-		return fmt.Errorf("failed to import armored key data into GnuPG keyring: %s", strings.TrimSpace(stderr.String()))
+		return fmt.Errorf("failed to import armored key data into GnuPG keyring: %s (stderr: %s)",
+			err, strings.TrimSpace(stderr.String()))
```

Comment:
```suggestion
		return fmt.Errorf("failed to import armored key data into GnuPG keyring: %s (%s)",
			strings.TrimSpace(stderr.String()), err)
```

would be better in this cause I think.

Reply:
Other option would be to do something like `<stderr>: <err>`.

Reply:
I made it one of:

- `failed to import armored key data into GnuPG keyring`
- `failed to import armored key data into GnuPG keyring: <stderr>`
- `failed to import armored key data into GnuPG keyring: <err>`
- `failed to import armored key data into GnuPG keyring: <stderr>: <err>`

In reality, when stderr is non-empty then err will likely also not be empty but now we're safe in the case where it is.